src/pages/notes/david-copperfield/index.mdx

```diff
@@ -19,22 +19,108 @@ import HowToCite from "/src/components/HowToCite.astro";
+
+<ImageWithCaption
+  src="/images/david_copperfield/DC_titlepage.jpg"
+  alt="Sample title page for David Copperfield"
+  caption="One of sixteen sheets where Dickens tested titles for David Copperfield, © Victoria & Albert Museum, London"
+/>
+
+Dickens’s correspondence with Forster throughout February 1849 shows his progression through several different title concepts. The first of these was “Mag’s Diversions / Being the personal history, experiences, and observation of / Mr David Mag the Younger, / of Blunderstone House.” Another iteration of “Mag’s Diversions” retained the central motif, but identified itself as being a personal history of “Mr David Copperfield the Younger / And his Great-Aunt Margaret.” This idea was later replaced by a series of prototype titles that brought the biographical element of the novel more sharply into focus, including “The Copperfield Disclosures,” “The Last Will and Testament of Mr David Copperfield,” “Copperfield’s Entire,” “Copperfield, Complete,” and, ultimately, “The Copperfield Survey of the World as it Rolled.” The last of these titles pleased Dickens the most, and it became the novel’s working title as he embarked upon the first installment (Forster 2.78-79). An undated handbill (Dexter 159) and two advertisements published in _The Examiner_ and the _Athenaeum_ in April present the novel under this title first (Burgis xxv), followed by the name that the novel would ultimately be published under: _The Personal History, Adventures, Experience, and Observation of David Copperfield the Younger of Blunderstone Rookery (Which He never meant to be Published on any Account)_. It was not until the second chapter had been drafted, according to Forster’s account, that Dickens had “defined to himself, more clearly than before, the character of the book,” and subsequently recognized the “propriety of rejecting everything not strictly personal from the name given to it” (Forster 2.79). Rather than a survey of the world, _Copperfield_ was to be the narration of a life, and it is significant that even through the many iterations of the novel’s title, all sixteen trial title sheets feature the words “Personal History.” 
+
+<ImageWithCaption
+  src="/images/david_copperfield/DC_johnson.jpg"
+  alt="Survey handbill for David Copperfield from the John Johnson Collection."
+  caption="Book Prospectus for David Copperfield, 1849. Copyright John Johnson Collection of Printed Ephemera, Bodleian Library, Oxford, Book Prospectus 12 (55; http://johnjohnson.chadwyck.co.uk/"
```

Comment:
Citation needs closing parentheses and also perhaps a direct URL to the website, I think